packages/vuetify/src/composables/theme.ts

```diff
@@ -0,0 +1,231 @@
+// Utilities
+import { capitalize, computed, inject, provide, ref, watch } from 'vue'
+import { useVuetify } from '@/framework'
+import { colorToInt, colorToRGB, contrastRatio } from '../util'
+
+// Types
+import type { InjectionKey, Ref } from 'vue'
+
+interface BaseColors {
+  background: string
```

Comment:
How about we have a naming convention similar to ${name}Bg ${name}Text. Because, after all, regardless of the naming, the text is likely going to be against another color. So, ´Bg', ´Text' would be required for any color name. alphaBg, alphaText, primaryBg, primaryText, etc.

Reply:
Text would be `onBackground`